access/types/forms.py

```diff
@@ -211,13 +211,14 @@ def add_table_fields(self, i, config, field_class, widget_class, multiple=False)
 
     def add_field(self, i, config, field_class, widget_class,
             initial=None, correct=None, choices=None, multiple=False,
-            widget_attrs={'class': 'form-control'}):
+            widget_attrs={'class': 'form-control'}, post_data = None):
```

Comment:
`post_data = None` should be `post_data=None`. No space is used with the keyword arguments.